fixtures/shehds/led-spot-60w.json

```diff
@@ -0,0 +1,487 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "LED Spot 60W",
+  "categories": ["Moving Head"],
+  "meta": {
+    "authors": ["yids"],
+    "createDate": "2022-11-08",
+    "lastModifyDate": "2022-11-08",
+    "importPlugin": {
+      "plugin": "qlcplus_4.12.1",
+      "date": "2022-11-08",
+      "comment": "created by Q Light Controller Plus (version 4.12.3)"
+    }
+  },
+  "links": {
+    "productPage": [
+      "https://www.shehds.com/products/good-technology-led-60w-gobos-moving-head-lighting-for-dmx512-stage-effect-dj-festivals-disco-home-entertainments-in-9-11-channels"
+    ],
+    "manual": [
+      "https://ueeshop.ly200-cdn.com/u_file/UPAG/UPAG904/2005/file/f56ea62fb9.pdf"
+    ],
+    "video": [
+      "https://www.youtube.com/watch?v=2ob8XVgcoRw"
+    ]
+  },
+  "physical": {
+    "dimensions": [19.8, 27.2, 14.4],
+    "weight": 3.2,
+    "power": 60,
+    "bulb": {
+      "type": "LED"
+    },
+    "DMXconnector": "3-pin"
+  },
+  "wheels": {
+    "Color wheel": {
+      "slots": [
+        {
+          "type": "Color",
+          "name": "White"
+        },
+        {
+          "type": "Color",
+          "name": "Red"
+        },
+        {
+          "type": "Color",
+          "name": "Orange"
+        },
+        {
+          "type": "Color",
+          "name": "Lemon Yellow"
+        },
+        {
+          "type": "Color",
+          "name": "Green"
+        },
+        {
+          "type": "Color",
+          "name": "Blue"
+        },
+        {
+          "type": "Color",
+          "name": "Pink"
+        },
+        {
+          "type": "Color",
+          "name": "Sky Blue"
+        },
+        {
+          "type": "Color",
+          "name": "Sky Blue & Pink"
+        },
+        {
+          "type": "Color",
+          "name": "Pink & Blue"
+        },
+        {
+          "type": "Color",
+          "name": "Blue & Green"
+        },
+        {
+          "type": "Color",
+          "name": "Green & Lemon Yellow"
+        },
+        {
+          "type": "Color",
+          "name": "Lemon Yellow & Orange"
+        },
+        {
+          "type": "Color",
+          "name": "Orange & Red"
+        },
+        {
+          "type": "Color",
+          "name": "Automatic"
```

Comment:
I'm also not sure what 'Automatic' is in this context. Is it an effect that cycles the wheel?

Reply:
@FloEdelmann I've corrected the colours, I'm just not quite sure how to express these automatic effects on the wheels.

Reply:
See my response below: https://github.com/OpenLightingProject/open-fixture-library/pull/2907#discussion_r1020884022